frontend/workflows/k8s/src/update-probe.tsx

```diff
@@ -0,0 +1,385 @@
+            textFieldLabels: {
+              disabledField: "Current Value",
+              updatedField: "New Value",
+            },
+            input: {
+              type: "string",
+              key:
+                probeType === "livenessProbe"
+                  ? `${containerBase}.livenessProbe.failureThreshold`
+                  : `${containerBase}.readinessProbe.failureThreshold`,
+              validation: string().matches(SECONDS_REGEX),
+            },
+          },
+        ]}
+      />
+      <ButtonGroup>
+        <Button text="Back" variant="neutral" onClick={() => onBack()} />
+        <Button text="Update" variant="destructive" onClick={onSubmit} />
+      </ButtonGroup>
+    </WizardStep>
+  );
+};
+
+const Confirm: React.FC<ConfirmChild> = () => {
+  const deployment = useDataLayout("deploymentData").displayValue() as IClutch.k8s.v1.Deployment;
+  const updateData = useDataLayout("updateData");
+  const currentDeploymentData = useDataLayout(
+    "currentDeploymentData"
+  ).displayValue() as IClutch.k8s.v1.Deployment;
+
+  const updateRows: any[] = [];
+
+  let updatedContainer = false;
+  let probeType = "livenessProbe";
+  deployment.deploymentSpec.template.spec.containers.forEach(container => {
+    if (!updatedContainer) {
+      updateRows.push({ name: "Container Name", value: container.name });
+      updatedContainer = true;
+    }
+    Object.keys(container.livenessProbe).forEach(livenessAttribute => {
+      const typeValue = typeof container.livenessProbe[livenessAttribute];
+      if (typeValue !== "object") {
+        const newValue = container.livenessProbe[livenessAttribute];
+        const oldValue = findContainer({
+          deploymentSpec: currentDeploymentData.deploymentSpec,
+          containerName: container.name,
+        }).livenessProbe[livenessAttribute];
+        if (newValue !== oldValue) {
+          updateRows.push({
+            name: `Old ${livenessAttribute}`,
+            value: oldValue,
+          });
+          updateRows.push({
+            name: `New ${livenessAttribute}`,
+            value: newValue,
+          });
+          probeType = "livenessProbe";
+        }
+      }
+    });
+    Object.keys(container.readinessProbe).forEach(readinessAttribute => {
+      const typeValue = typeof container.readinessProbe[readinessAttribute];
+      if (typeValue !== "object") {
+        const newValue = container.readinessProbe[readinessAttribute];
+        const oldValue = findContainer({
+          deploymentSpec: currentDeploymentData.deploymentSpec,
+          containerName: container.name,
+        }).readinessProbe[readinessAttribute];
+        if (newValue !== oldValue) {
+          updateRows.push({
+            name: `Old ${readinessAttribute}`,
+            value: oldValue,
+          });
+          updateRows.push({
+            name: `New ${readinessAttribute}`,
+            value: newValue,
+          });
+          probeType = "readnessProbe";
```

Comment:
should this be `readinessProbe`?
```suggestion
          probeType = "readinessProbe";
```

Reply:
Yes that's correct updated